MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -53,6 +56,8 @@ public override SpriteFontContent Process(FontDescription input, ContentProcesso
                         if (File.Exists(fontFile))
                             break;
                     }
+                    if (File.Exists(fontFile))
```

Comment:
side note, that if missing should have been the reason why font compilation was failing, are you sure the new native dependencies are needed?